src/pg_to_evalscript/javascript_common/common.js

```diff
@@ -44,3 +44,94 @@ function parse_rfc3339(dt, default_h = 0, default_m = 0, default_s = 0) {
 
   return result;
 }
+
+class ValidationError extends Error {
+  constructor({ name, message }) {
+    super(message);
+    this.name = name;
+  }
+}
+
+const VALIDATION_ERRORS = {
+  MISSING_PARAMETER: "MISSING_PARAMETER",
+  WRONG_TYPE: "WRONG_TYPE",
+  NOT_NULL: "NOT_NULL",
+  NOT_ARRAY: "NOT_ARRAY",
+  NOT_INTEGER: "NOT_INTEGER",
+  MIN_VALUE: "MIN_VALUE",
+  MAX_VALUE: "MAX_VALUE",
```

Comment:
To add in the future:
- `NOT_BOOLEAN`

FFTR

Reply:
Boolean was added